src/test/java/org/jenkinsci/plugins/gitclient/CliGitAPITempFileTest.java

```diff
@@ -10,6 +10,7 @@
 import java.util.Random;
 import static org.hamcrest.MatcherAssert.*;
 import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
```

Comment:
The `org.junit.Assert.assertThat` is intentionally not used since an assertThat is already provided by the wildcard imports from org.hamcrest.
```suggestion
```